packages/apollo-engine-reporting-protobuf/.gitignore

```diff
@@ -0,0 +1,2 @@
+protobuf.d.ts
```

Comment:
Yeah, I think it would make sense to be more explicit about the specific file you're excluding.

Reply:
I'm curious why `reports.proto` wouldn't move into this newly introduced `src` directory?

As a general principle, I find it strange that `protobuf.d.ts` (or anything) inside the `src/` tree is Git ignored **or** generated.  The previous implementation here didn't violate that expectation since everything was generated into `dist/`.